File: 119.cpp
class Solution {
public:
    vector<int> getRow(int rowIndex) {
        vector<vector<int>> vec;
        vec.push_back(vector<int>{1});
        for(int i=1; i<=rowIndex; i++){
            vector<int> tmp{1};
            for(int j=1; j<i; j++){
                tmp.push_back(vec[i-1][j-1] + vec[i-1][j]);
            }
            tmp.push_back(1);
            vec.push_back(tmp);
        }
        return vec[rowIndex];
    }
};

File: golang/119/119.go
package main

func getRow(rowIndex int) []int {
	trian := [][]int{}
	trian = append(trian, []int{1})
	if rowIndex == 0 {
		return trian[0]
	}
	for i := 1; i <= rowIndex; i++ {
		tmp := []int{1}
		for j := 1; j <= i-1; j++ {
			tmp = append(tmp, trian[i-1][j-1]+trian[i-1][j])
		}
		tmp = append(tmp, 1)
		trian = append(trian, tmp)
	}
	return trian[rowIndex]
}
